lib/count-users-by-year-for-deletion.rb

```diff
@@ -8,28 +8,27 @@
   config.retry = true
 end
 
-today = Date.today.next_day
-lastyear = today - 365
+lastyear = Date.today.next_day - 365
 
 puts "Users 2012 - 2018 to one year ago tomorrow, who may have tickets associated"
-puts @client.search(:query => "type:user created_at>2012-01-01 created_at<#{lastyear} role:end-user -name:Zendesk organization_id:none").count
+puts @client.search(:query => "type:user created_at>2012-01-01 created_at<#{lastyear} role:end-user -name:Zendesk organization_id:none").count.to_i
```

Comment:
Can we update the query to include 1st January?

```suggestion
puts @client.search(:query => "type:user created_at>=2012-01-01 created_at<#{lastyear} role:end-user -name:Zendesk organization_id:none").count.to_i
```

Reply:
@adityapahuja Please see comment above.

Reply:
Resolved as suggested.